config/schema.yaml

```diff
@@ -84,64 +83,60 @@ schema:
         type: string
         doc: >
           a string value composed only of letters, numbers, and underscores.
-      foreignkeys:
+      foreignKeys:
       -
         src: dataset_type_name
         tgt: DatasetType.dataset_type_name
       -
         src: run_id
-        tgt: run.execution_id
+        tgt: Run.execution_id
       -
         src: quantum_id
-        tgt: quantum.execution_id
+        tgt: Quantum.execution_id
+      -
+        src: abstract_filter
+        tgt: AbstractFilter.abstract_filter
       -
         src: camera
         tgt: Camera.camera
-        limited: false
```

Comment:
Are these removals an anomaly of the diff?

Reply:
No, they became unnecessary, because I've made it so Foreign Keys are automatically removed from a limited schema if either of their tables are.